includes/Admin/Admin_Page.php

```diff
@@ -0,0 +1,68 @@
+<?php
+/**
+ * Class WordPress\Plugin_Check\Admin\Admin_Page
+ *
+ * @package plugin-check
+ */
+
+namespace WordPress\Plugin_Check\Admin;
+
+/**
+ * Class is handling admin tools page functionality.
+ *
+ * @since n.e.x.t
+ */
+class Admin_Page {
+
+	/**
+	 * Initializes hooks.
+	 */
+	public function add_hooks() {
```

Comment:
This and the other methods in this class are missing `@since n.e.x.t` tags.

Reply:
Thanks @felixarntz I've updated the PR with the missing `@since` tags.